Assets/HoloToolkit/SpatialSound/Scripts/AudioInfluencerManager.cs

```diff
@@ -0,0 +1,242 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Class which supports IAudioInfluencers being used with audio sources.
+    /// </summary>
+    /// <remarks>
+    /// AudioInfluencerManager requires an AudioSource component. If one is not attached, it will be added automatically.
+    /// Each sound playing game object needs to have an AudioInfluencerManager attached in order to have it's audio influenced.
+    /// </remarks>
+    [DisallowMultipleComponent]
+    [RequireComponent(typeof(AudioSource))]
+    public class AudioInfluencerManager : MonoBehaviour
+    {
+        /// <summary>
+        /// Frequency below the nominal range of human hearing.
+        /// </summary>
+        /// <remarks>
+        /// This frequency can be used to set a high pass filter to allow all
+        /// audible frequencies through the filter.
+        /// </remarks>
+        public static readonly float NeutralLowFrequency = 10.0f;
```

Comment:
could this be const instead?

Reply:
if it's expected for devs to modify this value let's remove the readonly modifier.

Reply:
This is why I generally use static readonly instead of const: https://blogs.msdn.microsoft.com/davidklinems/2006/10/27/const-vs-static-readonly/

Devs are not expected to modify either of the neutral frequencies.